src/priority/priority.py

```diff
@@ -401,6 +410,9 @@ def unblock(self, stream_id):
 
         :param stream_id: The ID of the stream to unblock.
         """
+        if stream_id == 0:
+            raise MissingStreamError("Stream 0 is non-existent")
+
         # When a stream becomes unblocked,
```

Comment:
Yes, almost certainly. ;) I don't remember what I was trying to say, so let's just remove it.